app/middlewares/validators/users.js

```diff
@@ -21,3 +24,24 @@ exports.findAndDecryptPassword = async (req, _, next) => {
   }
   return next(errors.badRequestError('Wrong user or password'));
 };
+
+exports.verifyToken = (req, _, next) => {
+  try {
+    const {
+      common: {
+        session: { secret }
+      }
+    } = config;
+    const token = req.headers.authorization;
+
+    if (!token) {
+      return next(errors.loginError('Unauthorized user'));
+    }
+
+    jwt.verify(token, secret);
```

Comment:
The payload of the token could bee used in the controller, it is relevant to add it to the `req` object